developer_manual/core/theming.rst

```diff
@@ -26,6 +26,15 @@ All themes, whether copied or new, must meet two key criteria, these are:
 1. They must be located in a directory under the ``apps`` directory of your ownCloud installation.
 2. They require a configuration file called ``appinfo/info.xml`` to be present.
 
+.. note:: 
+   When copying the original theme supplied with ownCloud, make sure that you delete ``appinfo/signature.json``. If you don’t, when you make changes, an integrity warning will be triggered, as the original theme is `signed`_.
```

Comment:
I've made the two changes suggested. However, I don't see `appinfo/signature.json` in `apps/theme-example`.